pages/streams.rst

```diff
@@ -118,6 +118,46 @@ Alarm callbacks are a list of events that are being processed when an alert is t
 
 If the email alarm callback is used because it appears once or multiple times in the alarm callback list, or the alarm callback list is empty so the email transport is used per default, then the list of alert receivers is used to determine which recipients should receive the alert nofications. Every Graylog user (which has an email address configured in their account) or email address in that list gets a copy of the alerts sent out.
 
+Email Alert Callback
+====================
+
+The email alert callback can be used to send an email to the configured alert receivers when the conditions are triggered.
+
+Three configuration options are available for the alert callback to customize the email that will be sent.
+
+.. image:: /images/stream_alert_callback_email_form.png
+
+The *E-Mail Body* and *E-Mail Subject* are `jmte <https://code.google.com/p/jmte/>`_ templates. jmte is a minimal template engine that supports variables, loops and conditions. See the `jmte documentation <http://jmte.googlecode.com/svn/trunk/doc/index.html>`_ for a language reference.
+
+We expose the following objects to the templates.
+
+``stream``
+  The stream this alert belongs to.
+
+  * ``stream.id`` ID of the stream
+  * ``stream.title`` title of the stream
+  * ``stream.description`` stream description
+``stream_url``
+  A string that contains the HTTP URL to the stream.
+``check_result``
+  The check result object for this stream.
+
+  * ``check_result.triggeredCondition`` string representation of the triggered alert condition
+  * ``check_result.triggeredAt`` date when this condition was triggered
+  * ``check_result.resultDescription`` text that describes the check result
+``backlog``
+  List of message objects. Can be used to iterate over the messages via ``foreach``.
+
+  Each message in the list has the following fields.
```

Comment:
We should add a hint to the `message` object (and that the `toString()` representation is being used when not qualifying it further) so that this is consistent with the other variables (e. g. `stream` or `check_result`).